css/index.css

```diff
@@ -1,3 +1,367 @@
+
+  width: 691px;
+  vertical-align: top;
+  margin-top: 35px;
+}
+
+header nav a {
+
+  margin-right: 65px;
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+
+  margin-top: 25px;
+}
+
+/* cta */
+
+.cta .cta-text {
+
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  margin-top: 69px;
+}
+
+@media (max-width:800px) {
+  .cta img {
+    display: none;
+  }
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+@media (max-width:500px) {
+  .contact h4 {
+    font-size: 25px;
+  }
+
+  .contact p {
+    font-size: 20px;
+  }
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+@media (max-width:500px) {
+  footer p {
+    font-size:25px;
+    margin-bottom: 10px;
+  }
+}
+
+/*New code starts here */
+
+header {
+	display: flex;
+}
+
+@media (max-width:800px) {
+  header {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  header .logo {
+    order: -1;
+  }
+}
+
+@media (max-width:500px) {
+
+  header nav a {
+    font-size: 26px;
+    flex-direction: column;
+    display:flex;
+    align-items: center;
+    margin: 10px 0;
+    border-top: 1px solid black;
+    padding:17px 0;
+}
+
+  header nav .contact-link {
+    border-bottom: 1px solid black;
+  }
+
+}
+
+.cta {
+	display: flex;
+}
+
+@media (max-width:800px) {
+  .cta {
+    flex-direction: column;
+    align-items: center;
+  }
+}
+
+@media (max-width:500px) {
+  .cta {
+    margin-top: -66px;
+    margin-bottom: -35px;
+
+  }
+}
+
+.top-content {
+	display: flex;
+}
+
+@media (max-width:500px) {
+  .top-content{
+    flex-direction: column;
+    font-size: 18px;
+  }
+
+  .top-content .text-content {
+    margin-top: 46px;
+  }
+
+  .top-content .text-content p {
+    width:200%;
+    font-size:28px;
```

Comment:
after setting the main container correctly, you will want to remove this width: 200% here to make the paragraphs act like you want them to